R4.1/templates-community/kali.conf

```diff
@@ -0,0 +1,124 @@
+# vim: ft=make ts=4 sw=4
+
+GIT_BASEURL ?= https://github.com
+GIT_PREFIX ?= QubesOS/qubes-
+BRANCH ?= master
+
+RELEASE ?= 4.1
+
+VERBOSE = 2
+BACKEND_VMM=xen
+
+DIST_DOM0 ?= fc25
```

Comment:
May be this is not an issue, but I noticed that all the other R4.1 templates have `fc29` for `DIST_DOM0` ...

Reply:
In release builds this isn't an issue, as this config is used only for root.img part (where `DISTS_VM` matters), not packaging it into rpm (that would be done in `DIST_DOM0` environment). But lets make it consistent.
Thanks for noticing!

Reply:
Yes indeed, it's a mistake. A copy paste from R4.0 not totally reviewed :)